web/templates/attendance/index_coordinator.html.eex

```diff
@@ -32,8 +32,18 @@
             >
               <i class="material-icons">more_vert</i>
             </button>
-            <ul class="mdl-menu mdl-menu--bottom-right mdl-js-menu"
-                for="tr_menu_<%= offered_course.id %>"
+            <ul
+              class="
+                mdl-menu mdl-js-menu
+                <%=
+                  if index > 10 && index > length(@offered_courses)-4 do
```

Comment:
that's a good point, the whole thing could be extracted to a function `menu_position_class` or something like that